src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryArgumentCollection.java

```diff
@@ -150,14 +154,14 @@
          * The program FindBadGenomicKmersSpark can produce such a list for you.
          */
         @Argument(doc = "file containing ubiquitous kmer list. see FindBadGenomicKmersSpark to generate it.",
-                fullName = "kmersToIgnore")
+                fullName = "kmers-to-ignore")
         public String kmersToIgnoreFile;
 
         /**
          * This is a path to a text file of contig names (one per line) that will be ignored when looking for inter-contig pairs.
          */
         @Argument(doc = "file containing alt contig names that will be ignored when looking for inter-contig pairs",
-                fullName = "crossContigsToIgnore", optional = true)
+                fullName = "cross-contigs-to-ignore", optional = true)
         public String crossContigsToIgnoreFile;
 
         private static final String OUTPUT_ORDER_SHORT_NAME = "sort";
```

Comment:
I know this is not your committed change, but while we are at it, maybe change the next string const to "assembliesAlignmentsSortOrder"?